tests/python/pants_test/backend/python/test_interpreter_cache.py

```diff
@@ -29,10 +29,11 @@ def _make_bad_requirement(requirement):
 
     E.g. 'CPython==2.7.5' becomes 'CPython==99.7.5'
     """
-    if PY3:
-      return str(requirement).replace('==3', '==99')
-    else:
-      return str(requirement).replace('==2.', '==99')
+    return (
+      str(requirement).replace('==3', '==99')
+      if PY3
+      else str(requirement).replace('==2.', '==99')
+    )
```

Comment:
This is purely a refactor. Ternary expression better expresses the intent of the code block.

Reply:
I'm fine with this, but just be aware here you tread on unfirm ground. The intent of this is to return and so ternary (for example, not sure if this is what resonated with you) vs. Unreadable now since it needs parens to group multilinear is a debate with validity on both sides. This is where fairly comprehensive style guides come in handy to enforce a shared understanding of what (arbitrary) standard of readability or beauty to adopt. My personal rule for ternary is if it doesn't fit on an untortured (artificially making it fit by making variable names "too" short) 100 col single line, don't do it. But that's just my unjustifiable personal rubric.

Reply:
True, good point. The important benefit for me is that this turns it into an _expression_, rather than a _statement_. I have a strong bias towards finding expressions easier to reason about.

Either way, your comment prompted me to realize there's a way to deduplicate this all and to respond to your point. Should be good now. Thanks for the feedback!